packages/block-library/src/post-featured-image/block.json

```diff
@@ -1,6 +1,21 @@
 {
 	"name": "core/post-featured-image",
 	"category": "design",
+	"attributes": {
+		"isLink": {
+			"type": "boolean",
+			"default": false
+		},
+		"rel": {
+			"type": "string",
+			"attribute": "rel",
+			"default": ""
+		},
+		"linkTarget": {
+			"type": "string",
+			"default": "_blank"
```

Comment:
Why are we defaulting to `_blank`?

Reply:
Hm.. I guess it was natural to me as I've been expecting and prefer that approach, but after your comment and some research seems to be incorrect. Thanks for pointing this out Ari.

I have pushed the changes to default and changed PostTitle default as well.